Bump minimum required version of Python from 3.7 to 3.8 (#1465)
PlasmaPy/PlasmaPy: changelog/1465.trivial.1.rst

> An attempt to import PlasmaPy from a version of Python below the

Reviewer: ```suggestion
An attempt to import `plasmapy` from a version of Python below the
```

This a pkg reference instead of a project reference.  Importing the PlasmaPy project doesn't make sense.

Reviewer: This also doesn't also doesn't satisfy the tense outlined in the change log guide.  Maybe something more like...

```rst
Changed the raised exception to `ImportError` (from a general `Exception`)
when attempting to import `plasmapy` from a Python version below the
minimum supported version.
```

> An attempt to import `plasmapy` from a version of Python below the
> minimum supported value will now raise an `ImportError` instead of an
> `Exception`.

Reviewer: ```suggestion
Changed the raised exception to `ImportError` (from a general `Exception`)
when attempting to import `plasmapy` from a Python version below the
minimum supported version.
```

Just re-commenting this because the other suggestion was taken up quickly enough that the suggestion was likely resolved before this comment was viewed.

Again, tense in accordance with our changelog guide.
